docs/website/root/mithril/threat-model.md

```diff
@@ -0,0 +1,318 @@
+
+:::info To do
+
+Verification process being trustless, does it really make sense to include those in the Threat Model?
+
+:::
+
+#### Mithril certificate verification process
+
+Mithril clients download snapshots and verify associated certificates using the mithril-client library, either from a CLI tool or [embedded in a browser](https://mithril.network/explorer/) 
+
+* **availability**: Yes (?)
+* **confidentiality**: No
+* **integrity**: No (the snapshots and certificates are assumed to be secure, integrity is inherent to the protocol)
+
+#### Mithril snapshots & certificates
+
+Mithril certificates are produced by the aggregator from the individual signatures. Mithril certificates' security rests upon a chain of trust ultimately pointing at the Genesis certificate. Mithril aggregator maintains snapshots and certificates (aggregated multi-signatures) to be served to clients
+
+* **availability**: No (Snapshots and certificates are just a fallback, a cardano-node client can always retrieve the infromation from the cardano network itself, albeit much more slowly)
+* **confidentiality**: No
+* **integrity**: No (the snapshots and certificates are assumed to be secure, integrity is inherent to the protocol)
+
+#### Mithril Genesis Verification Key
+
+Mithril genesis verification key is stored in [GitHub](https://github.com/input-output-hk/mithril/blob/main/mithril-infra/configuration/release-mainnet/genesis.vkey) and only modifiable through an approved merged PR
+
+* **availability**: Yes (it's needed to verify a whole chain of certificate)
+* **confidentiality**: No
+* **integrity**: No (integrity is inherent to the protocol)
+
+#### Mithril Genesis Signing Key
+
+The corresponding signing key is stored in IOG's VaultWarden, and used only once, when the genesis certificate is generated. 
+
+* **availability**: No (? The key is not needed unless a re-genesis process is required, but then a new key could be used instead?)
+* **confidentiality**: Yes
+* **integrity**: Yes (?)
+
+## Threat & Mitigations
+
+### Resource exhaustion on Cardano relay
+
+- DoS of a `mithril-signer` running next to a `cardano-node` being a relay
+- Assets at risk: 
+   - [Block diffusion](#block-diffusion)
+   - [Mithril signatures diffusion](#mithril-signatures-diffusion)
+
+### Resource exhaustion on Cardano block producer
+
+* Assets at risk:
+  - [Block production](#block-production)
+
+### Block diffusion
+* Diffusion is ensured through the connection between BPs, local relays, and downstream/upstream relays
+* preventing them to operate can harm the Cardano network 
+* Relay hosts connect the BP to the network, 
+* Starving relay hosts prevents Mithril signatures and key registration to be shared
+* Starving a cardano-node running on a relay host would prevent or delay the diffusion of new blocks thus harming 
+* Compromising relay hosts would be an extreme form of starving resources
```

Comment:
Is this threat not the same as "Resource exhaustion on Cardano relay"?

Reply:
Yes, I moved this section as a sub-section of the "Resource exhaustion on Cardano relay" section :+1: